### PR TITLE
SOLR-16512 : Eliminate noggit JSONWriter.Writable from Solr classes

### PR DESCRIPTION
**Pull request by noblepaul**

### CODE REVIEW

**Comment by mkhludnev** [comment]
Seems fine. Couldn't find review button, btw.

**Comment by noblepaul** [comment]
> Seems fine. Couldn't find review button, btw.

I think if you go to the "files" tab , it's on the top right

File: solr/solrj/src/java/org/apache/solr/common/cloud/ClusterState.java [hunk 1]
```diff
@@ -295,15 +296,12 @@ private static DocCollection collectionFromObjects(
   }
 
   @Override
-  public void write(JSONWriter jsonWriter) {
-    LinkedHashMap<String, DocCollection> map = new LinkedHashMap<>();
-    for (Entry<String, CollectionRef> e : collectionStates.entrySet()) {
-      if (e.getValue().getClass() == CollectionRef.class) {
```
Comment (risdenk) [inline solr/solrj/src/java/org/apache/solr/common/cloud/ClusterState.java:301]: This if statement check was redundant before right?

**Review by risdenk — APPROVED** [review]

File: solr/solrj/src/java/org/apache/solr/common/cloud/ClusterState.java (hunk 1 above)
Comment (noblepaul) [reply solr/solrj/src/java/org/apache/solr/common/cloud/ClusterState.java:301]: yes. it was redundant. I could remove it

**Pull request merged**